coldfront/core/utils/management/commands/initial_setup.py

```diff
@@ -8,13 +8,31 @@
 
 
 class Command(BaseCommand):
+    help = 'Run setup script to initialize the Coldfront database'
+
+    def add_arguments(self, parser):
+         parser.add_argument("-fo", "--force_overwrite", help="Force intial_setup script to run with no warning.")
```

Comment:
Let's make this a boolean arg by adding `action="store_true"`. Also, let's rename the short flag to just `-f`.